lib/org/chromium/webidl/ast/Operation.js

```diff
@@ -23,6 +23,7 @@ foam.CLASS({
 
   methods: [
     function outputWebIDL(o) {
+      if (this.isStatic) o.outputStrs('static ');
```

Comment:
Similarly, static definitions do not seem to be able to have qualifiers (e.g. `getter`, `setter`, `deleter`), so the order of printing shouldn't matter.